docs/testing-operators.md

```diff
@@ -195,6 +195,16 @@ NAME                TYPE          ENDPOINT              REGISTRY   DISPLAYNAME
 johndoe-operators   appregistry   https://quay.io/cnr   johndoe                 Succeeded   The object has been successfully reconciled   30s
 ```
 
+Additionally, a `CatalogSource` is created in the `marketplace` namespace:
+
+```
+kubectl get catalogsource -n marketplace
+
+NAME                           NAME        TYPE   PUBLISHER   AGE
+johndoe-operators              Custom      grpc   Custom      3m32s
+[...]
+```
+
 ### 5. View Available Operators
 
 Once the `OperatorSource` is deployed, the following command can be used to list the available operators (until an operator is pushed into quay, this list will be empty):
```

Comment:
```suggestion
Once the `OperatorSource` and `CatalogSource` is deployed, the following command can be used to list the available operators (until an operator is pushed into quay, this list will be empty):
```

Reply:
...`OperatorSource` and `CatalogSource` **are** deployed...

Reply:
Done.